src/mongo/db/pipeline/expression.cpp

```diff
@@ -3144,8 +3144,18 @@ Value ExpressionPow::evaluate(const Document& root) const {
     }
 
     long long result = 1;
+
+    // When baseLong == -1 and expLong is < 0 the following for loop will never run because expLong
+    // will always be less than 0 so result will always be 1 but the result can potentialy be -1
+    // ex baselong = -1 expLong = -5  then result should be -1
+
```

Comment:
Remove extra newline, and on 3158. Generally we prefer to keep the comments directly above the line(s) of code they're referencing.